include/linux/if_link.h

```diff
@@ -359,8 +359,8 @@ enum {
 	IFLA_INFO_KIND,
 	IFLA_INFO_DATA,
 	IFLA_INFO_XSTATS,
-	IFLA_INFO_SLAVE_KIND,
-	IFLA_INFO_SLAVE_DATA,
+	IFLA_INFO_SECONDARY_KIND,
```

Comment:
I understand the sentiment and in general I agree.  FRR does not own this header file and we merely import it into the repo.  This type of change makes it very very hard for us maintainers to know when if_link.h is changed by the kernel.  The proper place for this change would be with the linux kernel and after you get that there we could change it here.